plugins/utils/flash.nix

```diff
@@ -245,19 +242,20 @@ in
           '';
 
           # Not sure what is the type...
-          prefix = helpers.defaultNullOpts.mkNullable (types.listOf types.anything) ''
-            [ ["⚡" "FlashPromptIcon"] ]
-          '' "";
-          winConfig = helpers.defaultNullOpts.mkNullable (types.attrsOf types.anything) ''
-            {
-              relative = "editor";
-              width = 1;
-              height = 1;
-              row = -1;
-              col = 0;
-              zindex = 1000;
-            }
-          '' "See nvim_open_win for more details";
+          prefix = helpers.defaultNullOpts.mkListOf types.anything [
+            [
+              "⚡"
+              "FlashPromptIcon"
+            ]
+          ] "";
+          winConfig = helpers.defaultNullOpts.mkAttrsOf types.anything {
+            relative = "editor";
+            width = 1;
+            height = 1;
+            row = -1;
+            col = 0;
+            zindex = 1000;
+          } "See nvim_open_win for more details";
```

Comment:
```suggestion
          } "See `:h nvim_open_win` for more details.";
```

Reply:
I'd prefer to avoid increasing the PR's scope by improving descriptions.

Reply:
Moved to #1714